practice-one/README.md

```diff
@@ -0,0 +1,27 @@
+# OVERVIEW
+
+* This document provides requirements, technical stack, and estimation for HTML CSS Practice.
+* Design: [figma](https://www.figma.com/file/EKwr6Bv1LsXdmDeo4FJMjo/Ride-sharing-web-ui---bootstrap-guide-lines-(Community)?type=design&node-id=36%3A54643&mode=design&t=pexqcgU7063mTSxG-1)
+
+# TIMELINE
+
+* 32 hours  (from 2023/11/16 - to 2023/11/23).
+
+# TEAM SIZE
+
+* One developer
+
+# TARGET
+
+* Understand HTML5/CSS3 properties and use them correctly
+* Slice from Figma to HTML/CSS
+* Understand the concepts of Flexbox
+* Cross-browser testing (Chrome, Firefox, and Safari latest version)
+
+# TECHNICAL STACK
+
+* HTML5/CSS3
+
+# TASK MANAGEMENT
+
+* [Trello](https://trello.com/b/u86xROVi/tiennguyen-internship-html-css)
```

Comment:
Missing `Requirement` section in this [document](https://docs.google.com/document/d/1OGwfL-4wypUoC5s6fV40w0kNR-wpbcokLH31edjIEzM/edit#heading=h.1t3h5sf)

Reply:
Beside, please add a `Getting Start` section to guide how to run your practice. You can refer the format below:

```
## GETTING START

* Clone repository

  ```
  git clone ....
  ```
* Checkout to branch `[BRANCH_NAME]`

  ```
  git checkout ...
  ```
* Move to folder `[FOLDER_NAME]`

  ```
  cd ....
  ```
* Install package

  ```
  npm install
  ```
* Run server

  ```
  npm start
  ```
```